compiler/damlc/daml-ide/src/DA/Daml/LanguageServer/Visualize.hs

```diff
@@ -0,0 +1,56 @@
+-- Copyright (c) 2019 The DAML Authors. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- | Execute Daml commands
+
+module DA.Daml.LanguageServer.Visualize
+    ( setCommandHandler
+    ) where
+
+import qualified Data.Aeson as Aeson
+import           Language.Haskell.LSP.Types
+import           Development.IDE.LSP.Server
+
+import Development.IDE.Types.Logger
+
+import qualified Data.Text as T
```

Comment:
I don’t understand the reasoning behind grouping and sorting the imports this way. Unless there is a good reason to split up imports (e.g. to differentiate between imports from external libraries and our own code), I prefer to just have a single block of imports sorted alphabetically by the module name.

Reply:
I was following the same style within the modules in that folder. I have no strong preference here. Will get rid of it